data_steward/retraction/retract_utils.py

```diff
@@ -79,6 +84,65 @@ def get_src_id(mapping_type):
     return src_id
 
 
+def get_datasets_list(project_id, dataset_ids):
+    # initialize list of all datasets in project
+    all_dataset_ids = [
+        dataset.dataset_id for dataset in bq.list_datasets(project_id)
+    ]
+
+    if not dataset_ids or dataset_ids == consts.NONE:
+        dataset_ids = []
+    elif dataset_ids == consts.ALL_DATASETS:
+        dataset_ids = all_dataset_ids
+    else:
+        dataset_ids = dataset_ids.split()
+        # only consider datasets that exist in the project
+        dataset_ids = [
+            dataset_id for dataset_id in dataset_ids
+            if dataset_id in all_dataset_ids
+        ]
+
+    # filter out datasets not of interest
```

Comment:
I would suggest adding a little more to this comment.  It's obvious we're filtering sandbox and staging datasets from the function names.  You might add something like we are only interested in datasets containing PPI or EHR data.